internal/hclogslog/adapter.go

```diff
@@ -0,0 +1,159 @@
+// Taken from https://github.com/evanphx/go-hclog-slog and modified to use
```

Comment:
I must say; I am not a fan of this...

Reply:
Also, why internal, why not vendored? 

Is the modification done by you or the author?

Reply:
Actually it is vendored, since it is here in the code and not pulled in as a dependency. It cannot be put in the vendor directory because it is modified.
It is modified by me; if it were modified by the author, I would pull the dependency instead :-)

If you explain why you are not a fan, I can give further explanation...

Reply:
See also explanation in previous comment about log/slog and the adapter.

Reply:
> If you explain why you are not a fan,

I think this is what is bothering me:
- inconsistent use of multiple pkgs to achieve the same thing. The same is true with my comment about the new imported test package.
- complexity of this new code: i need to understand this new code, just because I am adapting to the fact in point 1

> I can give further explanation...

no need to bother with this... just venting a bit... I just have the feeling we are importing two much stuff, that can be simplified and make understanding overall the cogito package unnecessarily more complex...

Reply:
Created some tickets in PCI backlog to cover this.